uptane-standard.md

```diff
@@ -868,32 +873,37 @@ In order to perform partial verification, an ECU SHALL perform the following ste
-3. Download and check the Root metadata file from the Director repository, following the procedure in {{check_root}}.
-4. Download and check the Timestamp metadata file from the Director repository, following the procedure in {{check_timestamp}}.
-5. Download and check the Snapshot metadata file from the Director repository, following the procedure in {{check_snapshot}}.
-6. Download and check the Targets metadata file from the Director repository, following the procedure in {{check_targets}}.
-7. Download and check the Root metadata file from the Image repository, following the procedure in {{check_root}}.
-8. Download and check the Timestamp metadata file from the Image repository, following the procedure in {{check_timestamp}}.
-9. Download and check the Snapshot metadata file from the Image repository, following the procedure in {{check_snapshot}}.
-10. Download and check the top-level Targets metadata file from the Image repository, following the procedure in {{check_targets}}.
-11. Verify that Targets metadata from the Director and Image repositories match. A primary ECU MUST perform this check on metadata for all images listed in the Targets metadata file from the Director repository downloaded in step 6. A secondary ECU MAY elect to perform this check only on the metadata for the image it will install. (That is, the target metadata from the Director that contains the ECU identifier of the current ECU.) To check that the metadata for an image matches, complete the following procedure:
-    1. Locate and download a Targets metadata file from the Image repository that contains an image with exactly the same file name listed in the Director metadata, following the procedure in {{resolve_delegations}}.
-    2. Check that the Targets metadata from the Image repository matches the Targets metadata from the Director repository:
-        1. Check that the non-custom metadata (i.e., length and hashes) of the unencrypted image are the same in both sets of metadata.
-        2. Check that all "MUST match" custom metadata (e.g., hardware identifier and release counter) are the same in both sets of metadata.
-        3. Check that the release counter in the previous targets metadata file is less than or equal to the release counter in this targets metadata file.
+1. Load the repository mapping metadata ({{repo_mapping_meta}}).
+2. Load the latest verified time from the time server, if implemented.
+3. Refresh the top-level metadata for the Director repository, by following the steps in {#refresh_toplevel_metadata}, for the first repository in the list of repositories in the mapping metadata.
+4. If this ECU is a primary ECU, then iterate through the image metadata listed in the Director repository's top-level Targets metadata, and **store** all image metadata.  The primary ECU SHOULD also ensure that the image metadata from the director repository doesn't contain any ECU identifiers for ECUs not actually present in the vehicle.
```

Comment:
What does "**store** all image metadata" mean?

Reply:
I think it's clear which metadata is intended, right? (If not, LMK!)  So the question is just what does storing it mean.  I wondered if this would be clear....

The operations on this metadata all refer to this metadata as "**stored** metadata" in bold, mostly for editors.  As we iterate through the repositories in the mapping, we note what each repository lists as the appropriate image metadata for a given image (whether it comes from the top-level Targets role of the Director or some delegated role in some image repository).  Once we have all of those, we compare them to each other, and discard them if they don't match in the appropriate ways, or return them if they do.

The point of noting these as "stored metadata" here is just to keep clear what bundle we're talking about.  For example, we are not comparing all top-level Targets metadata from all repositories, and we are not discarding the metadata roles from the repositories if this comparison fails.

How that is actually done does not matter to us at this level, I think.

So is the intention clear enough, does it seem correct, and do you think the text should be altered to make it clearer?

Reply:
I'm not sure the boldface is helping here; in my opinion it just makes it a bit confusing why the word "store" is so important.

Also, "doesn't" -> "does not" for maximum formality. :)